CHANGELOG.md

```diff
@@ -93,8 +104,6 @@
 - Added documentation pages for `Index` and its APIs.
 - Added support for `DataFrame.assign`.
 - Added support for `DataFrame.stack`.
-- Added support for `DataFrame.pivot` and `pd.pivot`.
```

Comment:
Oh, @sfc-gh-jkew sorry, actually, snowpark python seems use the release tag commit hash here https://github.com/snowflakedb/snowpark-python/releases/tag/v1.19.0, what you had before seems correct.

Reply:
hmm. We may need to ask for help. Originally I was diffing with the release tag; but I think diffing with the branch still makes sense like you said in your first comment.